ui/voice/src/arrows.ts

```diff
@@ -9,54 +10,73 @@ export const brushes = new Map<string, DrawBrush>([
   ['pink', { key: 'vpn', color: '#ee2080', opacity: 0.5, lineWidth: 12 }],
   ['yellow', { key: 'vyl', color: '#ffef00', opacity: 0.6, lineWidth: 12 }],
   ['orange', { key: 'vor', color: '#f6931f', opacity: 0.8, lineWidth: 12 }],
-  ['white', { key: 'vwh', color: '#ffffff', opacity: 1.0, lineWidth: 12 }],
   ['brown', { key: 'vgy', color: '#7b3c13', opacity: 0.8, lineWidth: 12 }],
-  ['red', { key: 'vrd', color: '#881010', opacity: 0.8, lineWidth: 12 }],
   ['grey', { key: 'vgr', color: '#666666', opacity: 0.8, lineWidth: 12 }],
+  ['red', { key: 'vrd', color: '#881010', opacity: 0.8, lineWidth: 12 }],
+  ['white', { key: 'vwh', color: '#ffffff', opacity: 1.0, lineWidth: 15 }],
 ]);
 
 const LABEL_SIZE = 40; // size of arrow labels in svg user units, 100 is the width of a board square
 
-export function numberedArrows(choices: [string, Uci][], asWhite: boolean, timer?: number): DrawShape[] {
+export function numberedArrows(choices: [string, Uci][], timer: number | undefined, asWhite: boolean): DrawShape[] {
   if (!choices) return [];
   const shapes: DrawShape[] = [];
   const dests = new Map<Key, number | Set<number>>();
+  const preferred = choices[0][0] === 'yes' ? choices.shift()?.[1] : undefined;
   choices.forEach(([, uci]) => {
-    shapes.push({ orig: src(uci), dest: dest(uci), brush: `v-grey` });
+    shapes.push({ orig: src(uci), dest: dest(uci), brush: `v-grey`, modifiers: { hilite: uci === preferred } });
```

Comment:
chessground 8.3.8 `DrawModifiers` has no `hilite`

Reply:
found it there https://github.com/lichess-org/chessground/pull/263 and published 8.3.9